conans/client/command.py

```diff
@@ -854,6 +856,16 @@ def remove(self, *args):
             self._cache.remove_locks()
             self._user_io.out.info("Cache locks removed")
             return
+        elif args.system_reqs:
```

Comment:
If we are not supporting (I don't want it) `-p` with `--system-reqs` we should raise an error when used together. And test it.

Reply:
oh boy, I just pushed a new commit with package id support ... let's revert it.

Reply:
Okay, I just reverted the commit now. My intention to include package id support: https://github.com/conan-io/conan/pull/4354#issuecomment-456371582

Also, I've added `-p` check and a test to verify